src/pages/Collaborate/index.js

```diff
@@ -58,18 +58,7 @@ const Collaborate = () => {
               mainHeading='Volunteer With Us'
               subHeading='Help us improve the Civic Tech Index.'
               title='Read instructions on how to join Hack for LA'
-              description={[
-                'If you would like to volunteer for our organization please visit this ',
-                <Link
-                  aria-label='Read instructions on how to join Hack for LA'
-                  key='join'
-                  target='_blank'
-                  title='Read instructions on how to join Hack for LA'
-                  to='https://www.hackforla.org/join'
-                >
-                  page.
-                </Link>,
-              ]}
+              description='If you would like to volunteer for our organization please follow the link below for details.'
```

Comment:
Small nit here, 'organization' needs a comma
```suggestion
              description='If you would like to volunteer for our organization, please follow the link below for details.'
```